datasets/xsum/xsum.py

```diff
@@ -73,39 +96,59 @@ def _info(self):
     def _split_generators(self, dl_manager):
         """Returns SplitGenerators."""
 
-        dl_path = dl_manager.download_and_extract(_URL)
+        data_path = dl_manager.download_and_extract(_URL_DATA)
+        splits_path = dl_manager.download(_URL_SPLITS)
+
+        with open(splits_path, "r") as f:
```

Comment:
Could you specify the encoding each time you call `open` ? `encoding="utf-8"` for example.
This is for windows users.